dual_net.py

```diff
@@ -298,17 +297,18 @@ def input_fn():
     if steps is None:
         steps = EXAMPLES_PER_GENERATION // FLAGS.train_batch_size
     estimator.train(input_fn, hooks=[
-                        update_ratio_hook, step_counter_hook], steps=steps)
+        update_ratio_hook, step_counter_hook], steps=steps)
 
 
 def validate(working_dir, tf_records, checkpoint_name=None, validate_name=None):
     estimator = get_estimator(working_dir)
+    validate_name = validate_name or "selfplay"
```

Comment:
it's weird that this has different style than the next line.

can you unify them to
A = A or default
or
if B is none: B = default

Reply:
Done.